guides/user/plugins/index.md

```diff
@@ -123,6 +123,8 @@ See the command [reference](/reference/halyard/commands/#hal-plugins-repository)
 
 ## Add a plugin using Halyard
 
+>Note: When Halyard adds a plugin to a Spinnaker installation, it adds the plugin repository information to each service. This means each service  downloads the plugin and checks if an extension exists for that service. Each service restarts, which is not good for large Spinnaker installations. See the [Plugin configuration without Halyard](#plugin-configuration-without-halyard) section for how to avoid each service restarting.
```

Comment:
```suggestion
>Note: When Halyard adds a plugin to a Spinnaker installation, it adds the plugin repository information to each service. This means each service downloads the plugin and checks if an extension exists for that service. Each service restarts, which is not good for large Spinnaker installations. See [Plugin configuration without Halyard](#plugin-configuration-without-halyard) for how to avoid each service restarting.
```

Reply:
"...which is not good for large..." is a value judgment, and probably an accurate one, but can we describe the specific consquences?

Reply:
I admit I ripped off that sentence from one of the engineers. I'll check.

Reply:
So... the engineers said that service restart times in a large installation are loooooong. Clouddriver can take an hour or more if it has lots of accounts configured. They also said that the community is actively working to shorten service restart times.